src/popup/sass/components/_confirm_popup.scss

```diff
@@ -0,0 +1,75 @@
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 10; /* Sit on top */
+  padding-top: 140px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 63%;
+  height:56%;
+}
+.modal-close
+{
+  float: right;
+  font-size: 20px;
+  font-weight: bold;
+  position: relative;
+  top: -50px;
+  left:13px;
+  cursor: pointer;
+}
+.modal-header{
+    background-image: linear-gradient(to right, #ff4e14, #ff7d31);
+    margin-top: 0;
+    text-align:center;
+    padding: 5px;
+}
+.modal-text
+{
+  padding: 20px 5px 20px 5px;
+}
+.modal-button--cancel
+{
+  position: relative;
+  text-align: center;
+  background-color:#ee5b32;
+  font-size: 20px;
+  color: white;
+  top:10px;
+  left:60px;
+  margin: 0px 2px;
+  display: inline-block;
+  text-decoration: none;
+  border:1px solid #ee5b32;
+  cursor: pointer;
+  padding:4px 8px;
+}
+.modal-button--confirm
+{
+  position: relative;
+  text-align: center;
+  background-color:#ee5b32;
+  font-size: 20px;
+  color: white;
+  top:10px;
+  left:180px;
+  margin: 0px 2px;
+  display: inline-block;
+  text-decoration: none;
+  border:1px solid #ee5b32; 
+  cursor: pointer;
+  padding:4px 8px;
+}
```

Comment:
- Change the name of the file to _modal.scss because that's the name this component is being referenced as everywhere.
- This file looks like just another CSS file as not a single feature of SASS has been used. We can at least nest the rules so that redundant property can be reused.

Reply:
Should I make it as plain CSS file.

Reply:
No. Make use of SASS features like nested rules to refactor the stylesheet